### PR TITLE
Update charts to 1.2.0 release

### PR DESCRIPTION
**Pull request by ChandanSagar**
- Updated README
    - Updated chart versioin 1.2.0
    - Updated values.yaml
    - Updated ENV variables for openebs api server
    - Add tolerations for ndm daemonset
    - Add cstorpoolclusters rules in validatingWebhookConfigurations
    
    Signed-off-by: ChandanSagar <chandan.pradhan@mayadata.io>

### CODE REVIEW

**Comment by ChandanSagar** [comment]
@guangbochen : Please take a look.

**Comment by ChandanSagar** [comment]
@guangbochen Any update on this PR.

**Comment by guangbochen** [comment]
Will check later today, sorry was busy those days, cheers.

File: charts/openebs/latest/templates/daemonset-ndm.yaml [hunk 1]
```diff
@@ -8,6 +8,8 @@ metadata:
     release: {{ .Release.Name }}
     heritage: {{ .Release.Service }}
     component: ndm
+    openebs.io/component-name: ndm
```
Comment (guangbochen) [inline charts/openebs/latest/templates/daemonset-ndm.yaml]: can we use `apps/v1` of the DaemonSet for k8s v1.16 capability support.

Comment (ChandanSagar) [reply charts/openebs/latest/templates/daemonset-ndm.yaml]: Made a change.

**Comment by guangbochen** [comment]
LGTM and merged.

**Pull request merged**